lib/coverband/collectors/coverage.rb

```diff
@@ -57,7 +57,7 @@ def report_coverage
       rescue StandardError => e
         @logger&.error 'coverage failed to store'
         @logger&.error "Coverband Error: #{e.inspect} #{e.message}"
-        @logger&.error e.backtrace if @verbose
+        e.backtrace.each { |line| @logger&.error line } if @verbose
```

Comment:
could do ` @logger&.error e.backtrace.join(', ') if @verbose` but I guess either is fine

Reply:
Reason i like this approach is if someone is prefixing each log with something like the request id or job id, then each line of the backtrace which come in a search by that id.